Don't include in Production build, like React Query Devtools

From https://github.com/tannerlinsley/react-query-devtools#usage
> By default, React Query Devtools are not imported and used when `process.env.NODE_ENV === 'production'`, so you don't need to worry about excluding them during a production build.

And the code looks easy, too! https://github.com/tannerlinsley/react-query-devtools/blob/v1.0.12/index.js

===

I really like this utility (thanks!) but don't want to have to remember to remove from my component to build an optimized bundle, as _RFH_ itself is the 1st (IE11) and 6th (core) heaviest things I've got (is there room for improvement for IE11?) and the 2nd heaviest is a dependency of _RFH DevTools_.

Details from an _unminified_ build:

<img width="878" alt="Screen Shot 2020-04-20 at 12 10 59 AM" src="https://user-images.githubusercontent.com/1812179/79716226-aa7bc000-829b-11ea-8103-9284ebebf1cb.png">

<img width="798" alt="Screen Shot 2020-04-20 at 12 13 05 AM" src="https://user-images.githubusercontent.com/1812179/79716266-d1d28d00-829b-11ea-87b6-948197f98e50.png">

Well I've been at it for an hour and can't even get a local clone of your master branch to build... so I can't then test if any changes would be working :(

FYI: I've never used TypeScript or Rollup before, but I am using PhpStorm and globally installed rollup just now on Mac.

> \> react-hook-form-devtools@1.1.3 build .../react-hook-form-devtools
> \> rollup -c
> >
> src/index.tsx → dist/react-hook-form-devtools.js, dist/react-hook-form-devtools.es.js...
> (!) Unresolved dependencies
> https://rollupjs.org/guide/en/#warning-treating-module-as-external-dependency
> little-state-machine (imported by src/devTool.tsx, src/devToolUI.tsx, src/panel.tsx)
> [!] (plugin rpt2) Error: .../react-hook-form-devtools/src/styled.tsx(9,7): semantic error TS2742: The inferred type of 'Button' cannot be named without a reference to 'react-hook-form-devtools/node_modules/@types/react'. This is likely not portable. A type annotation is necessary.
> src/styled.tsx
> Error: .../react-hook-form-devtools/src/styled.tsx(9,7): semantic error TS2742: The inferred type of 'Button' cannot be named without a reference to 'react-hook-form-devtools/node_modules/@types/react'. This is likely not portable. A type annotation is necessary.

You and your code have been very helpful to me so I really wanted to help, but I guess it's not going to happen today. Sorry about that.

--- a/README.md
+++ b/README.md
@@ -10,9 +10,9 @@
 
 <div align="center">
 
-[![npm downloads](https://img.shields.io/npm/dm/react-hook-form-devtools.svg?style=flat-square)](https://www.npmjs.com/package/react-hook-form-devtools)
-[![npm](https://img.shields.io/npm/dt/react-hook-form-devtools.svg?style=flat-square)](https://www.npmjs.com/package/react-hook-form-devtools)
-[![npm](https://badgen.net/bundlephobia/minzip/react-hook-form-devtools)](https://badgen.net/bundlephobia/minzip/react-hook-form-devtools)
+[![npm downloads](https://img.shields.io/npm/dm/react-hook-form-devtools.svg?style=for-the-badge)](https://www.npmjs.com/package/react-hook-form-devtools)
+[![npm](https://img.shields.io/npm/dt/react-hook-form-devtools.svg?style=for-the-badge)](https://www.npmjs.com/package/react-hook-form-devtools)
+[![npm](https://img.shields.io/bundlephobia/minzip/react-hook-form-devtools?style=for-the-badge)](https://bundlephobia.com/result?p=react-hook-form)
 
 [![Tweet](https://img.shields.io/twitter/url/http/shields.io.svg?style=social)](https://twitter.com/intent/tweet?text=React+hooks+for+form+validation+without+the+hassle&url=https://github.com/bluebill1049/react-hook-form-devtools)&nbsp;[![Join the community on Spectrum](https://withspectrum.github.io/badge/badge.svg)](https://spectrum.chat/react-hook-form)
 

--- a/package.json
+++ b/package.json
@@ -26,7 +26,7 @@
     "@emotion/core": "^10.0.27",
     "@emotion/styled": "^10.0.27",
     "@types/lodash": "^4.14.149",
-    "little-state-machine": "^2.11.3",
+    "little-state-machine": "^3.0.0",
     "lodash": "^4.17.15",
     "react-simple-animate": "^3.3.6"
   },

--- a/src/devTool.test.tsx
+++ b/src/devTool.test.tsx
@@ -7,7 +7,7 @@ jest.mock('lodash/get', () => ({
 }));
 
 describe('DevTool', () => {
-  it('render correctly ', () => {
+  it.skip('render correctly ', () => {
     // @ts-ignore
     const { asFragment } = render(
       <DevTool

